test/e2e/common/pods.go

```diff
@@ -848,4 +853,119 @@ var _ = framework.KubeDescribe("Pods", func() {
 		validatePodReadiness(false)
 
 	})
+
+	ginkgo.It("should run through the lifecycle of Pods and PodStatus", func() {
+		podResource := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "pods"}
+		testNs := f.Namespace.Name
+		testPodName := "pod-test"
+		testPodImage := "nginx"
+		testPodImage2 := "httpd"
+		testPodLabels := map[string]string{"test-pod-static": "true"}
+		testPodLabelsFlat := "test-pod-static=true"
+
+		testPod := v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   testPodName,
+				Labels: testPodLabels,
+			},
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{
+					{
+						Name:  testPodName,
+						Image: testPodImage,
+					},
+				},
+			},
+		}
+		ginkgo.By("creating a Pod with a static label")
+		_, err := f.ClientSet.CoreV1().Pods(testNs).Create(context.TODO(), &testPod, metav1.CreateOptions{})
+		framework.ExpectNoError(err, "failed to create Pod")
```

Comment:
This (and most of the `ExpectNoError` calls in here) are examples of difficult to debug log messages.  Context that I can search for in the log is helpful.

eg: which pod? which namespace? ...

Reply:
Updated in cf24d1c8e1f